charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -241,6 +241,7 @@ spec:
           imagePullPolicy: {{ default .Values.image.pullPolicy .Values.sidecars.attacher.image.pullPolicy }}
           args:
             - --csi-address=$(ADDRESS)
+            - --timeout=60s
```

Comment:
Why not make it configurable?

Reply:
This parameter is configurable via `additionalArgs`.